package.json

```diff
@@ -41,17 +52,25 @@
     "grunt-ts": "^6.0.0-beta.16",
     "grunt-tslint": "^5.0.1",
     "grunt-webpack": "^3.0.2",
+    "html-webpack-include-assets-plugin": "^1.0.2",
+    "html-webpack-plugin": "^2.30.1",
     "intern": "~4.1.0",
+    "json-css-module-loader": "^1.0.2",
+    "optimize-css-assets-webpack-plugin": "^3.2.0",
     "postcss": "5.2.17",
     "postcss-cssnext": "2.10.0",
     "postcss-import": "9.1.0",
+    "postcss-loader": "^2.0.9",
     "postcss-modules": "0.6.4",
+    "resolve-url-loader": "^2.2.0",
     "sinon": "^4.1.2",
+    "source-map-loader-cli": "0.0.1",
     "ts-node": "^3.3.0",
     "tslint": "^5.8.0",
     "typed-css-modules": "^0.3.1",
     "typescript": "~2.6.2",
-    "webpack": "^3.7.1"
+    "webpack": "^3.9.1",
+    "webpack-dev-server": "^2.9.5"
   },
   "dependencies": {
     "@dojo/core": "~0.3.0",
```

Comment:
We should be able to move all of our dependencies to dev dependencies if our distribution is a webpack bundle.